client/src/components/relatedProducts/card.jsx

```diff
@@ -88,18 +104,33 @@ const Card = (props) => {
     }
   }
 
+  const handleClick = (event) => {
+    console.log(props.info)
+    props.handleChangeProduct(props.info)
```

Comment:
semi colon

Reply:
Also, consider deconstructing the props at the top of the function so that you don't have to continually refer to "props."